packages/esm-form-entry-app/src/app/fe-wrapper/fe-wrapper.component.css

```diff
@@ -2,6 +2,64 @@
   color: var(--omrs-color-danger);
 }
 
+.error-tile {
+  width: 60%;
+  margin: 1.25rem auto;
+  display: flex;
+  flex-flow: column wrap;
+  align-items: center;
+  justify-content: center;
+  padding: 1.5rem 0 1.75rem;
+}
+
+/* Tablet */
+:host-context(.omrs-breakpoint-lt-desktop) .error-tile {
+  background-color: white;
+}
+
+.heading {
+  font-size: 1rem;
+  font-weight: 600;
+  color: #161616;
```

Comment:
Please use color variables if they're available

Reply:
Not sure we can in this Angular context

Reply:
Oh weird! Feel free to merge, but what's the error you see when trying to use a color variable? Either the SCSS `@include` version or the CSS `var` version?

Reply:
I hadn't realized I could just change the stylesheet to a SASS file. Done.